spec/features/all_feature_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'pg'
+
+feature 'shows chitter peeps' do
+  scenario '.all shows peeps' do
```

Comment:
This is a feature test, so you're not testing the user journey, rather than a specific method.